browser/skus/skus_service_factory.cc

```diff
@@ -73,7 +74,10 @@ KeyedService* SkusServiceFactory::BuildServiceInstanceFor(
   return new skus::SkusServiceImpl(
       g_browser_process->local_state(),
       context->GetDefaultStoragePartition()
-          ->GetURLLoaderFactoryForBrowserProcess());
+          ->GetURLLoaderFactoryForBrowserProcess(),
+      base::ThreadPool::CreateSingleThreadTaskRunner(
```

Comment:
why are you passing in the task runner and current sequence to the constructor?

Reply:
Moved to just initialization